lib/identity_cache/fallback_fetcher.rb

```diff
@@ -11,7 +11,9 @@ def write(key, value)
     end
 
     def delete(key)
-      @cache_backend.delete(key)
+      result = @cache_backend.delete(key)
```

Comment:
The fallback fetcher is just for compatibility for cache backends that don't support CAS operations.  [memcached_store](https://rubygems.org/gems/memcached_store) is the only active support cache store that I know of that supports CAS.

The difference is that now it will just unconditionally log `[IdentityCache] delete for #{key}` with debug logging and won't attempt to indicate whether it failed or not.